doc/translations.md

```diff
@@ -70,6 +70,12 @@ If you only want to translate friendica into another language you wont need any
 
 For further information see the utils/README file.
```

Comment:
As the tools were moved from the `utils` directory to some other place, the documentation should not point to the README file in the outdated location. Instead the documentation from there should be integrated into the translation documentation ;-)